test/uk/gov/hmrc/economiccrimelevyreturns/controllers/actions/DataRetrievalActionSpec.scala

```diff
@@ -41,13 +43,14 @@ class DataRetrievalActionSpec extends SpecBase {
   }
 
   "transform" should {
-    "transform an AuthorisedRequest into a ReturnDataRequest" in {
-      when(mockEclReturnService.getOrCreateReturn(any())(any())).thenReturn(Future(emptyReturn))
+    "transform an AuthorisedRequest into a ReturnDataRequest" in forAll {
+      eclReturn: EclReturn => (internalId: String, eclReferenceNumber: String) =>
+        when(mockEclReturnService.getOrCreateReturn(any())(any())).thenReturn(Future(eclReturn))
 
-      val result: Future[ReturnDataRequest[AnyContentAsEmpty.type]] =
-        dataRetrievalAction.transform(AuthorisedRequest(fakeRequest, internalId))
+        val result: Future[ReturnDataRequest[AnyContentAsEmpty.type]] =
+          dataRetrievalAction.transform(AuthorisedRequest(fakeRequest, internalId, eclReferenceNumber))
```

Comment:
As per config discussion just now, should we rename all occurrences of eclReferenceNumber to eclRegistrationNumber?